src/core/Core.cpp

```diff
@@ -1,9 +1,34 @@
-#include "core/Core.hpp"
+#include "core/Core.hpp"
+// #include "common/product.hpp"

```

Comment:
In general, if you base on some commit it is better to create `pull request` on that commit/branch rather than comment sth out.